docs/src/apps/jsonrpc-api.md

```diff
@@ -137,7 +137,12 @@ Returns all information associated with the account of provided Pubkey
 #### Parameters:
 
 * `<string>` - Pubkey of account to query, as base-58 encoded string
-* `<object>` - (optional) [Commitment](jsonrpc-api.md#configuring-state-commitment)
+* `<object>` - (optional) Configuration object containing the following optional fields:
+  * (optional) [Commitment](jsonrpc-api.md#configuring-state-commitment)
+  * (optional) `encoding: <string>` - filter results by account type; currently supported: `circulating|nonCirculating`
+  encoding for Account data, either "binary" or json".
+    JSON encoding attempts to use program-specific state parsers to return more human-readable and explicit account state data.
```

Comment:
Document the fallback for an account with an unknown parser (encoding falls back to "binary").   Also let's document that the when using JSON, they should dynamically detect the fallback based on the string datatype of the returned data as we will add additional account parsers in the future